docs/xocto/development.md

```diff
@@ -78,6 +78,7 @@ Commit these changes in a single commit with subject matching
 After merging the pull request, push an annotated tag to Github with:
 
 ```sh
+make
```

Comment:
🧀 The [makefile](https://github.com/octoenergy/xocto/blob/main/makefile#L36) will pick up the _installed_ version of the package, which might be different to what's being developed.